### PR TITLE
added previous judges section

### PR DESCRIPTION
**Pull request by douglasawh**
also moved judges into that section

### CODE REVIEW

**Comment by douglasawh** [comment]
this still needs <h3>Dominic Francesco</h3> added, but I think it makes life easier to just merge this and then I can add him later

File: content/judges.yaml [hunk 1]
```diff
@@ -8,22 +8,41 @@ content:
     <h3>Sam Austin</h3>
     <p class="left max">I was introduced to the world of Creative Commons music back in 2010 after doing some quick google searching for free legal music. I stumbled across Jamendo and quickly fell in love with certain artists and this started a massive interest in finding bands that release music under cc licensing. Most of the artists and releases that I listen to lately have been released on a net label. Some are self released. I chose the Europe region because the majority of my favorite music discoveries from the recent decade have been from European artists. I hope that this world of cc music will last for many years.</p>
 
+   <h3>Kevin Binder</h3>
```
Comment (michaelGregoire) [inline content/judges.yaml]: @douglasawh You need to fix the indenting in this. Yaml is very dependent on proper indenting. Use space based tabbing, too. The build is currently broken as it is right now.

Comment (michaelGregoire) [reply content/judges.yaml]: Screenshot 2023-05-18 at 7 38 59 PM
The above is from the build log. It shows the first occurrence is misaligned indenting.

File: content/judges.yaml [hunk 2]
```diff
@@ -8,22 +8,41 @@ content:
...
     <h3>Nick Burgess</h3>
 
     <h3>Donnie Ozone</h3>
     <p><img src="/img/donnie-ozone.jpg" class="judge" /></p>
     <p class="left max">I am a Creative Commons friendly hip hop artist who resides in Brooklyn, NY. I started uploading my vocal recordings under Creative Commons licenses to various websites in 2012. Since then, my vocals have been featured in productions in various genres from hip hop to electronic dance music. I release CC licensed tracks via the blocSonic netlabel and I contribute a track to Netlabel Day every year. I am the CC CMA judge for the Oceania region.</p>
 
+
+    <h3>Jessica Harrison</h3>
+
+   <h3>Eddie Palmer</h3>
```
Comment (michaelGregoire) [inline content/judges.yaml]: Another indenting that will likely break the build.

**Comment by douglasawh** [comment]
@michaelGregoire I didn't change any of the yaml. I had a suspicion that the app didn't like me adding a `<h2>` tag for the previous judges, so I changed that but it still fails. 

None of the HTML is ahead of the yaml body tag

Does vercel have a particular line number it says is at issue?

**Comment by michaelGregoire** [comment]
@douglasawh It's currently failing due to end of life of node v14. I'll pull this branch and make the required change.

**Review by michaelGregoire — APPROVED** [review]

**Pull request merged**